kloppy/domain/models/common.py

```diff
@@ -224,7 +224,17 @@ class Metadata:
     provider: Provider
 
 
+class DatasetType(Enum):
```

Comment:
Same naming convention as `EventType`.  There is however `DatasetType = TypeVar("DatasetType")` defined in line 118 of `kloppy/helpers.py` as well as `kloppy/domain/services/transformers/__init__.py`.  In the latter case, it's class scoped.  Could be confusing. Necessary to rename?

Reply:
Good you kept the naming convention! 

The code in `TypeVar` is used for typehinting. Would be good indeed to rename it. Let's use `DatasetT` for `helpers.py` and the transformers

Reply:
Renamed.